autonomy/deploy/generators/kubernetes/templates.py

```diff
@@ -281,6 +281,19 @@
         - name: nodes
           persistentVolumeClaim:
             claimName: 'nodes'
+        - emptyDir: {{}}
+          name: local-tendermint
+      initContainers:
+        - name: copy-tendermint-configuration
+          image: "ubuntu:20.04"
+          command: ["bash", "-c"]
+          args: 
+          - "while [ ! -d /tendermint/node{validator_ix} ]; do sleep 1; done; cp -r /tendermint/node{validator_ix}/* /tm/"
```

Comment:
wait until the file is available, otherwise we instantly complete initialisation.

Reply:
@angrybayblade i think this is just explaining what is going on. @8ball030 can confirm.